wiki/Article_styling_criteria/Writing/de.md

```diff
@@ -230,23 +239,68 @@ Manche Wörter haben Varianten. Ihre bevorzugte Schreibweise ist wie folgt:
 - `erstellt` anstatt `beatmapped` oder `mapped`.
 - `BN` oder `Beatmap Nominators`, wenn auf die *Beatmap Nominators* verwiesen wird.
 
-### Wettbewerbsartikel
+### Turnierartikel
 
 Diese Worte müssen wie folgt verwendet werden (der Abstand muss übereinstimmen):
 
 - `NoMods`, `NoMod` oder `NM`
 - `FreeMods`, `FreeMod` oder `FM`
 - `Tiebreaker`
 
-## Abkürzungen, Akronyme und Initialwörter
-
-Abkürzungen, Akronyme und Initialwörter müssen bei ihrem ersten Auftreten vollständig ausgeschrieben werden. Nachfolgende Vorkommen sind optional, aber werden nur ausgeschrieben, sofern erforderlich. Das Folgende ist ein Beispiel:
+### Übersetzungen für häufige Begriffe
+
+Dieser Abschnitt listet häufig verwendete Begriffe und ihre entsprechenden Übersetzungen auf. Beim Übersetzen von Artikeln sollte darauf geachtet werden, möglichst die hier aufgelisteten Übersetzungen zu verwenden. Ausnahmen können gemacht werden, wenn die ursprüngliche Bedeutung durch die Übersetzung verloren geht.
+
+#### Gameplay
+
+| Englisch | Deutsch |
+| :-: | :-- |
+| `game modifier (mod)` | `Spielmodifikation (Mod)` |
+| `gameplay` | `Spielverlauf` |
+| `multiplier` | `Multiplikator` |
+| `success rate` | `Erfolgsrate` |
```

Comment:
My concern wasn't which would be more correct (in-game it's "pass rate", in the extended results screen, and on the web it's "success rate" in the beatmap info page), but that this is literally only used once in the wiki, and I don't see any other reason for including it here.

```suggestion
```